_extras/guide.md

```diff
@@ -274,6 +281,8 @@ their breath after what is often a frustrating couple of hours.
 
 ## [Citation]({{ page.root }}/12-citation/)
 
+We teach about citation because it is important to give learners the tools to respect the licensing chosen for the code.
```

Comment:
I think mentioning licensing here (especially because it comes right after the licensing lesson) is a little confusing.

Reply:
@munkm Since this was starting to go stale, I resolved the conflicts with the recent style update, and made your suggested changes to the content.  Let me know what you think.